index.js

```diff
@@ -1288,6 +1537,13 @@ instance.prototype.init_feedback = function () {
 				default: '0',
 				choices: self.CHOICES_PAGES,
 			},
+			{
```

Comment:
variables in feedbacks doesnt work currently. we arent tracking what variables are used properly, so unless we invalidate all of these whenever any variable changes, they wont update when the variables change